src/apps/Hosting/Files.js

```diff
@@ -89,12 +120,12 @@ const HostingFilesView = React.createClass({
         <InnerToolbar title="Website Hosting">
           <Show if={items.length && !isLoading}>
             <RaisedButton
-              label={isDefaultHosting ? 'Published' : 'Publish'}
-              onTouchTap={this.handleShowPublishDialog}
+              label="Open in tab"
```

Comment:
but what do we open with that button? shouldn't it say something like _Go to site_ or something like this?